app/controllers/ApplicationController.php

```diff
@@ -14,7 +14,7 @@ function __construct(ApplicationForm $applicationForm, SettingRepository $settin
   {
     $this->applicationForm = $applicationForm;
 
-    $this->beforeFilter('currentUser', ['only' => ['edit', 'update']]);
+    $this->beforeFilter('isOwnerOrAdmin', ['only' => ['edit', 'update', 'show']]);
```

Comment:
we have a filter for owner and we have a filter for admin, but for this we want either the owner or an admin to be able to view the page. You can apply multiple filters, but then they ALL have to be true. What I found was that if you want to have ORs, you have to make your own filter to handle that